app/components/draft_scheduled_post_header/draft_scheduled_post_header.tsx

```diff
@@ -64,23 +75,54 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
         displayName: {
             color: changeOpacity(theme.centerChannelColor, 0.64),
             ...typography('Body', 75, 'SemiBold'),
+            flexShrink: 1,
+            flexGrow: 1,
         },
         time: {
             color: changeOpacity(theme.centerChannelColor, 0.64),
             ...typography('Body', 75),
         },
+        scheduledContainer: {
+            display: 'flex',
```

Comment:
I see a bunch of these everywhere, this is not needed as is the default display on RN, ok to keep but just mentioning that is not needed.

Reply:
REmoved it.